modules/ROOT/pages/understanding-orchestration-using-mule.adoc

```diff
@@ -0,0 +1,62 @@
+= Understanding Orchestration Using Mule
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: anypoint studio, studio, mule esb, orchestration
+
+Orchestration is how Mule facilitates the process of moving data through a Mule implementation based on flows. Orchestration can be seen as combining service calls to create higher-level, more useful composite services, and implies implementing business-level processes combining business-specific services across applications and information systems.
```

Comment:
Orchestration is a process by which Mule apps facilitate the movement of data
through processors within Mule flows. Orchestration can be understood as
a combination of service calls that work together to create higher-level,
more useful, composite services that implement business-level processes. These
processes combine business-specific services across applications and
information systems.

Reply:
ok